cmd/merge.go

```diff
@@ -7,13 +7,19 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/Clever/microplane/initialize"
 	"github.com/Clever/microplane/merge"
 	"github.com/Clever/microplane/push"
 	"github.com/spf13/cobra"
 )
 
+var mergeFlagThrottle string
+var mergeThrottle string
+
+var mergeLimiter = time.NewTicker(1 * time.Millisecond)
```

Comment:
add comments to these three. Maybe put them in a block
```golang
// comment
var (
   merge.....
)
```

Reply:
added comments